lib/dpul_collections/solr.ex

```diff
@@ -322,11 +323,8 @@ defmodule DpulCollections.Solr do
   def add(docs, index \\ Index.read_index())
 
   def add(docs, index) when is_list(docs) do
```

Comment:
these both have the same implementation now? should we just have one of them?

Reply:
Yeah, I was kinda going with like...if I wrote this as a library, what would it be. Might be premature 🐣